geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -88,9 +88,18 @@ <h3>Transcription</h3>
             {% for chunk in document.digital_editions.0.content.html %}
                 <div class="img"></div> {# empty image div needed for grid layout #}
                 <div class="transcription-panel">
-                    {% if forloop.first %}<h3>Transcription</h3>{% endif %}
-                    {# display transcription in chunks based on loop index #}
-                    <div class="transcription">{{ chunk|h1_to_h3|safe }}</div>
+                    {% if forloop.first %}
+                        <h3>Transcription</h3>
+                        <span data-transcription-target="editionFullLabel" class="current-transcription">{{ document.digital_editions.0.display|safe }}</span>
+                    {% endif %}
```

Comment:
oh, I see – we missed the stimulus target because we currently have two different ways we output the transcription content, depending on whether there are images or not